libraries/Microsoft.Bot.Builder.Ai.Translation/LocaleConverterMiddleware.cs

```diff
@@ -1,94 +1,75 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+
+using Microsoft.Bot.Schema;
 using System;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Bot.Schema;
 
 namespace Microsoft.Bot.Builder.Ai.Translation
 {
     /// <summary>
-    /// Middleware that translates from the input locale to a specified locale.
+    /// Middleware to convert messages between different locales specified
```

Comment:
The older comment seems more accurate. Is this a merge change with recent work that Jonathan did?